src/pages/introduction/scholarly.mdx

```diff
@@ -4,4 +4,90 @@ title: "SCHOLARLY INTRODUCTION: Dickens’s Working Notes"
 description: "Charles Dickens’s Working Notes are ..."
 ---
 
-TODO
+import ImageWithCaption from "/src/components/ImageWithCaption.astro";
+import TOC from "/src/components/TOC.astro";
+
+{/* prettier-ignore */}
+<TOC>
+ - [Rethinking Dickens's Serial Form with the DDNP](#rethinking-dickens's-serial-form-with-the-ddnp)
```

Comment:
You'll need to remove the apostrophe from the # tag link. When Astro creates the ID for the header, it simply lowercases, strips punctuation, and concatenates with hyphens.